src/mpyl/schema/project.schema.yml

```diff
@@ -86,6 +86,7 @@ properties:
           - Renew Lets Encrypt Deploy
           - Helm Deploy
           - Dagster Deploy
+          - BPMN Diagram Deploy
```

Comment:
I've only seen only `bpm` used until now, what is `bpmn`?

Edit: also in this pr I see both being used, seems a bit confusing?

Reply:
bpmn specifically means the notation and diagram itself, bpm is a more generic term for methodology. You are right we should keep it consistent. We are using bpm more in our project. I will replace all bpmn to just bpm